cmd/root.go

```diff
@@ -66,6 +68,55 @@ func defaultConfig() string {
 	return filepath.Join(cfgDir, "bomctl", "bomctl.yaml")
 }
 
+func getDocumentCompletionArgs(
```

Comment:
Can we put this into its own file like `completion.go`? For the name, something simple like `func completions(...)` should be adequate. At the very least, let's leave off `get` since it isn't conventional in go